ament_cmake_version/cmake/ament_bump_development_version_if_necessary.cmake

```diff
@@ -0,0 +1,55 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#
+# Bump the exported package version to the passed version if the package
+# version in the manifest is lower.
+#
+# It is recommended to append the suffix ``-dev`` to the passed upcoming
+# version number.
+# If the package version in the manifest is equal or newer than the passed
+# development version this function call becomes a no op.
+#
+# .. note:: It is indirectly calling``ament_package_xml()`` if that hasn't
+#   happened already.
+#
+# :param target: development_version
+# :type target: string
+#
+# @public
+#
+macro(ament_bump_development_version_if_necessary development_version)
```

Comment:
My only comments are that `bump` doesn't seem right since they are specifying the full version here, and that I'd be more specific, like perhaps: `ament_set_development_version_if_current_version_less_than(<dev_version>)`. I know that's a mouthful, but I generally prefer something more specific even if it is longer to type.

Reply:
> My only comments are that `bump` doesn't seem right since they are specifying the full version here

Agreed.

> `ament_set_development_version_if_current_version_less_than`

I would like to see the word "export" in the name since the resulting version number is being exported to downsteam packages (as for all the other functions with the same term in the name).

Also "current version" is imo not specific enough since it is unclear that "current" likely means the version from the manifest?

Maybe: `ament_export_development_version_if_higher_than_manifest()`?

Reply:
`ament_export_development_version_if_higher_than_manifest()` sounds good to me.

Reply:
Renaming done in d63439434b24a46c3fea0fc6ece14a9e8ffed948. I also added a note about multiple invocations.